Formula/gstreamer.rb

```diff
@@ -21,51 +37,221 @@ class Gstreamer < Formula
     sha256 x86_64_linux:   "cf16cedfa1fbf8c2176bec39158bfed3a0c58ee4b92e7c88cd9ca50a422aaf5e"
   end
 
+  head do
+    url "https://gitlab.freedesktop.org/gstreamer/gstreamer.git", branch: "main"
+
+    resource "rs" do
+      url "https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs.git", branch: "main"
+    end
+  end
+
   depends_on "bison" => :build
+  depends_on "cargo-c" => :build
   depends_on "gobject-introspection" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "yasm" => :build
+  depends_on "cairo"
+  depends_on "dav1d"
+  depends_on "faac"
+  depends_on "faad2"
+  depends_on "fdk-aac"
+  depends_on "ffmpeg"
+  depends_on "flac"
   depends_on "gettext"
   depends_on "glib"
+  depends_on "glib-networking"
+  depends_on "graphene"
+  depends_on "gtk+3"
+  depends_on "gtk4"
+  depends_on "jpeg-turbo"
+  depends_on "json-glib"
+  depends_on "lame"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libpthread-stubs"
+  depends_on "libshout"
+  depends_on "libsoup"
+  depends_on "libusrsctp"
```

Comment:
This provides only a static library so there is no linkage. Will fix that separately.